docs/source/workshops/bioc-2015.rst

```diff
@@ -45,114 +32,87 @@ Install gcloud
+(2) Click on *"User guides, package vignettes and other documentation."*
 
-  * To run line-by-line, put your cursor on the desired line  and click the "Run" button or use keyboard shortcuts for Windows/Linux: ``Ctrl+Enter`` and Mac: ``Command+Enter``.
+(3) Click on vignette *"Bioc2015Workshop"* and follow the instructions there to run the vignettes **line-by-line** or **chunk-by-chunk**!
 
-  * To run chunk-by-chunk, put your cursor in the desired chunk and click the "Chunks -> Run Current Chuck" button. or use keyboard shortcuts for Windows/Linux: ``Ctrl+Alt+C`` and Mac: ``Command+Option+C``.
+  * To run **line-by-line**, put your cursor on the desired line  and click the "Run" button or use keyboard shortcuts for Windows/Linux: ``Ctrl+Enter`` and Mac: ``Command+Enter``.
 
-  * You can view the rendered analysis here `AllModalitiesDemo.md <https://github.com/googlegenomics/codelabs/blob/master/R/1000Genomes-BRCA1-analysis/AllModalitiesDemo.md>`_.
+  * To run **chunk-by-chunk**, put your cursor in the desired chunk and click the "Chunks -> Run Current Chuck" button. or use keyboard shortcuts for Windows/Linux: ``Ctrl+Alt+C`` and Mac: ``Command+Option+C``.
 
   .. image:: /_static/run_rmarkdown.png
     :alt: Run Rmarkdown
 
-Run the "Quality Control using Google Genomics" Codelab
---------------------------------------------------------
-
-In `Quality Control using Google Genomics`_ we perform common quality control checks such as singleton rate, transition/transversion rate, and Hardy-Weinberg Equilibrium on the :doc:`/use_cases/discover_public_data/platinum_genomes` gVCF variants.
-
-(1) Open the first codelab RMarkdown file.  From the RStudio *Files Pane*:
+Looking for more to try?
+-----------------
+* Try these samples on different datasets :doc:`/use_cases/discover_public_data/index`.
+* Find more example BigQuery queries in:
 
-  R -> PlatinumGenomes-QC
+  * `getting-started-bigquery`_
+  * `bigquery-examples`_
+  * `Quality Control using Google Genomics`_
 
-  Open ``Data-Overview.Rmd``
+* Run a `Google Cloud Dataflow`_ pipeline:
 
-(2) Set your current working directory.  From the RStudio *Files Pane*:
+  * :doc:`/use_cases/compute_identity_by_state/index`
+  * :doc:`/use_cases/compute_principal_coordinate_analysis/1-way-pca`
 
-  More -> Set As Working Directory
+Wrap up
+-------------
 
-(3) Start running `Data-Overview.Rmd <https://github.com/googlegenomics/codelabs/blob/master/R/PlatinumGenomes-QC/Data-Overview.Rmd>`_ line by line or chunk by chunk!
+"Stop" or "Delete" your VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  You can view the rendered analysis here `Data-Overview.md <https://github.com/googlegenomics/codelabs/blob/master/R/PlatinumGenomes-QC/Data-Overview.md>`_.
+(1) Go to the Google Developers Console: https://console.developers.google.com/project/_/compute/instances
```

Comment:
Since you are using a "deep link" here, this should probably say " and select your project"